src/preact/test-utils.js

```diff
@@ -0,0 +1,55 @@
+import * as preact from /*OK*/ 'preact';
+
+import {isTest} from '#core/mode';
+
+const rafs = [];
+/**
+ * @param {*} cb
+ * @return {number}
+ */
+function flushableRaf(cb) {
+  cb.__completed = false;
```

Comment:
Weakset?

Reply:
Good idea. Cleaner than adding a key to all of the funcs. Done